src/Protocol/DFRN.php

```diff
@@ -1291,31 +1294,13 @@ public static function deliver($owner, $contact, $atom, $dissolve = false)
 
 			switch ($rino_remote_version) {
 				case 1:
+				case 2:
```

Comment:
It's for the transition between new nodes with no RINO2 and legacy/stable nodes that still are advertising a RINO version 2. This code ensures that the following communication will be using the version 1 no matter what instead of nothing at all.